suite-native/transactions/package.json

```diff
@@ -17,6 +17,7 @@
         "@suite-common/wallet-core": "*",
         "@suite-common/wallet-types": "*",
         "@suite-common/wallet-utils": "*",
+        "@suite-native/accounts": "*",
```

Comment:
I think this can be removed because it's unused now

Reply:
[df99815](https://github.com/trezor/trezor-suite/pull/6319/commits/df99815d2c384b0067e0574d83b832ed98253fc7)